assets/components/src/with-wizard-screen/style.scss

```diff
@@ -176,3 +176,54 @@
 		}
 	}
 }
+
+// Details
+
+.newspack-details {
+	margin: 32px 0;
+
+	ul {
+		list-style: disc;
+		margin: 0;
```

Comment:
This makes the webhooks list a bit cramped:

<img width="567" alt="image" src="https://user-images.githubusercontent.com/7383192/134152862-a8e4f73b-9e96-4d10-a17c-0d7866c15ecc.png">

Reply:
Fixed: https://github.com/Automattic/newspack-plugin/pull/1114/commits/89d785105ba5bb31206f27c3012ce88183c10f44